src/utils/input_validation/validators.py

```diff
@@ -0,0 +1,177 @@
+import os, sys, uuid 
+
+def validate_room_number(person_id: str) -> bool: 
```

Comment:
Variable name is a little confusing, I assume you mean room_id or something similar?